pennylane/capture/make_plxpr.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The make_plxpr function and helper methods"""
+
+from typing import Sequence, Union
+
+has_jax = True
+try:
+    import jax
+except ImportError:
+    has_jax = False
+
+
+def make_plxpr(circuit: "qml.QNode", static_argnums: Union[int, Sequence[int]] = (), **kwargs):
+    """Takes a QNode and returns a function that, when called, produces a PLxPR representing
+    the QNode with the given args.
+
+    This function relies on `jax.make_jaxpr` as part of creating the representation. Any
+    keyword arguments passed to `make_plxpr` that are not directly used in the function will
+    be passed to make_jaxpr.
+
+    Args:
+        circuit (QNode):  the QNode to be captured
+
+    Kwargs:
+        static_argnums (Union(int, Sequence[int])): optional, an int or collection of ints
+            that specify which positional arguments to treat as static (trace- and compile-time constant).
+        **kwargs: any additional arguements specifically for jax.make_jaxpr
+
+    Returns:
+        Callable: function that, when called, returns the PLxPR representation of `circuit` for the specified inputs.
+
+
+    ** Example **
```

Comment:
I _think_ this won't render with spaces.
```suggestion
    **Example**
```

Reply:
Ahh, you're right! I made the same mistake in a different PR yesterday and it definitely doesn't render correctly.